README.md

```diff
@@ -1,76 +1,44 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> One paragraph statement about the project.
+> First project module one
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![screenshot](./app.png)
 
 ## Built With
 
 - Major languages
-- Frameworks
-- Technologies used
+HTML
+CSS
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
+[Live Demo Link](https://obote.github.io/Hello-Microverse1/)
 
 ### Setup
-
+Configure linters 
 ### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+VS Code or any text editor
 
 
 ## Authors
 
 👤 **Author1**
```

Comment:
**OPTIONAL**
- [ ] You can improve the `Author1` section of your ReadMe by replacing the second author with your name.